### ISSUE TITLE
Not activated when opening via command line

### ISSUE THREAD

**Issue by shaicoleman** [jfcherng/Sublime-AutoSetSyntax#16]
When opening a file through the command line when sublime is closed, the plugin isn't activated.

To reproduce:
* Close sublime if open
* Create `example.test`
* Configure AutoSetSyntax
* Run `subl example.test` - the syntax isn't applied automatically
* File > Revert - the syntax is applied

**example.test**
```
#!/usr/bin/env ruby3
puts 'This is a test'
```

**AutoSetSyntax.sublime-settings**
```
{
    "run_on_startup_views": true,
    "user_syntax_rules": [
        {
            "syntaxes": ["scope:source.ruby"],
            "selector": "text.plain",
            "rules": [
                {
                    "constraint": "first_line_contains_regex",
                    "args": ["#!.*\\b.?ruby.?\\b"]
                }
            ]
        }
    ]
}
```

**Versions:**
AutoSetSyntax v2.10.7
Sublime build 4143
Linux Mint 21.1 (based on Ubuntu 22.04)

**Comment by jfcherng** [comment]
Thanks. This should've been fixed in ef333e04cde47211be4cf3194140ad803097d5bf and released as `4114-2.11.0`, which will be available within hours.

**Comment by jfcherng** [comment]
Btw, you may use the `is_interpreter` too. But yeah regex is generally more powerful.

```json
        {
            "syntaxes": ["scope:source.ruby"],
            "selector": "text.plain",
            "rules": [
                {
                    "constraint": "is_interpreter",
                    "args": ["ruby"],
                    "kwargs": {"loosy_version": true} // will also match "ruby3"
                }
            ]
        }
```
